internal/config/operations_test.go

```diff
@@ -0,0 +1,135 @@
+	}
+	if len(op.AdditionalProperties) != 0 {
+		t.Errorf("Expected empty additional properties map but got %v properties", len(op.AdditionalProperties))
+	}
+
+	// Test for AnnotateNamespace
+	op, exists = dev[AnnotateNamespace]
+	if !exists {
+		t.Errorf("Operation %v not found in operations map", AnnotateNamespace)
+	}
+	if op.Type != int32(meshes.OpCategory_CONFIGURE) {
+		t.Errorf("Expected operation type %v but got %v", int32(meshes.OpCategory_CONFIGURE), op.Type)
+	}
+	if op.Description != "Annotate Namespace" {
+		t.Errorf("Expected operation description %v but got %v", "Annotate Namespace", op.Description)
+	}
+
+	// Test for JaegerAddon
+	op, exists = dev[JaegerAddon]
+	if !exists {
+		t.Errorf("Operation %v not found in operations map", JaegerAddon)
+	}
+	if op.Type != int32(meshes.OpCategory_CONFIGURE) {
+		t.Errorf("Expected operation type %v but got %v", int32(meshes.OpCategory_CONFIGURE), op.Type)
+	}
+	if op.Description != "Add-on: Jaeger" {
+		t.Errorf("Expected operation description %v but got %v", "Add-on: Jaeger", op.Description)
+	}
+	if len(op.AdditionalProperties) != 3 {
+		t.Errorf("Expected 3 additional properties but got %v properties", len(op.AdditionalProperties))
+	}
+	if op.AdditionalProperties[ServiceName] != "jaeger" {
+		t.Errorf("Expected %v for %v additional property but got %v", "jaeger", ServiceName, op.AdditionalProperties[ServiceName])
+	}
+	if op.AdditionalProperties[ServicePatchFile] != "file://templates/oam/patches/service-loadbalancer.json" {
+		t.Errorf("Expected %v for %v additional property but got %v", "file://templates/oam/patches/service-loadbalancer.json", ServicePatchFile, op.AdditionalProperties[ServicePatchFile])
+	}
+	if op.AdditionalProperties[HelmChartURL] != "https://helm.linkerd.io/stable/linkerd-jaeger-30.4.5.tgz" {
+		t.Errorf("Expected %v for %v additional property but got %v", "https://helm.linkerd.io/stable/linkerd-jaeger-30.4.5.tgz", HelmChartURL, op.AdditionalProperties[HelmChartURL])
+	}
+
+	// Test for MultiClusterAddon
+	op, exists = dev[MultiClusterAddon]
+	if !exists {
+		t.Errorf("Operation %v not found in operations map", MultiClusterAddon)
+	}
+	if op.Type != int32(meshes.OpCategory_CONFIGURE) {
+		t.Errorf("Expected operation type %v but got %v", int32(meshes.OpCategory_CONFIGURE), op.Type)
+	}
+	if op.Description != "Add-on: Multi-cluster" {
+		t.Errorf("Expected operation description %v but got %v", "Add-on: Multi-cluster", op.Description)
+	}
+	if len(op.AdditionalProperties) != 3 {
+		t.Errorf("Expected 3 additional properties but got %v properties", len(op.AdditionalProperties))
+	}
+	if op.AdditionalProperties[ServiceName] != "linkerd-gateway" {
+		t.Errorf("Expected %v for %v additional property but got %v", "linkerd-gateway", ServiceName, op.AdditionalProperties[ServiceName])
+	}
+	if op.AdditionalProperties[ServicePatchFile] != "file://templates/oam/patches/service-loadbalancer.json" {
```

Comment:
We have a [PR](https://github.com/meshery/meshery-linkerd/pull/495) for removing OAM in the future. So, we do not need to care about the part which was related to OAM. Meshery was drop OAM support already.

Reply:
So Shall I directly remove the OAM lines?

Reply:
Yes, let us remove this.